CMakeLists.txt

```diff
@@ -108,10 +109,32 @@ if(c-ares_FOUND)
   message(STATUS "c-ares found!")
   target_link_libraries(${PROJECT_NAME} PRIVATE c-ares_lib)
   set(TRANTOR_SOURCES ${TRANTOR_SOURCES} trantor/net/inner/AresResolver.cc)
-else()
+else(c-ares_FOUND)
   set(TRANTOR_SOURCES ${TRANTOR_SOURCES} trantor/net/inner/NormalResolver.cc)
+endif(c-ares_FOUND)
+
+if ((CMAKE_GENERATOR MATCHES "Visual Studio") OR (CMAKE_GENERATOR MATCHES "Xcode"))
```

Comment:
Can we just remove the check of IDE and set all headers as a part of TRANTOR_SOURCES?

Reply:
Well, I just wanted not to touch the project when not generating for non-IDE, since they are not needed then.
But it you want, I'll change it today.

Reply:
@marty1885 What do u think about this?

Reply:
I don't get why the detection is needed. (My guess is they didn't show up in the IDEs' source menu?). 

I agree, let's make the build script simpler.